src/components/Radio/Radio.js

```diff
@@ -38,12 +41,11 @@ export default class Radio extends React.Component {
       .forEach((_, i) => {
         if (this.props[`option${i}`]) {
           const uuid = `${this.state.id}-${i}`;
-          const inline = this.props.inline ? '-inline' : '';
 
           radiosToRender.push(
-            <div className={`chi-form__item ${inline}`} key={`radio-${i}`}>
-              <div className="chi-radio">
-                <input className="chi-radio__input" type="radio" name={`radios-${uuid}`} id={uuid}
+            <div className={this.props.inline ? `${FORM_CLASSES.ITEM} ${INLINE_CLASS}` : 'chi-col -w--12 -mb--1'} key={`radio-${i}`}>
```

Comment:
We have already defined some of these utility classes.

```suggestion
            <div className={this.props.inline ? `${FORM_CLASSES.ITEM} ${INLINE_CLASS}` : `${UTILITY_CLASSES.COLUMN} ${UTILITY_CLASSES.MARGIN.BOTTOM[1]} -w--12`} key={`radio-${i}`}>
```